CONTRIBUTORS.md

```diff
@@ -7,35 +7,35 @@ Chapel release:
 Contributors to the current release
 -----------------------------------
 * Ben Albrecht, [HPE]
-* Himanshu Bhadani, individual contributor
 * Akshansh Bhanjana, individual contributor
+* Ankush Bhardwaj, individual contributor
 * Paul Cassella, [HPE]
 * [Brad Chamberlain], [HPE]
-* Sanket Chaudhari, individual contributor
-* Anway De, individual contributor
+* Soohoon Choi, [HPE]
 * Garvit Dewan, [GSoC 2019] student from [Indian Institute of Technology Roorkee]
 * Krishna Kumar Dey, [GSoC 2019] student from [Indian Institute of Information Technology, Sri City] (former individual contributor)
 * Lydia Duncan, [HPE]
+* Omar Elawady, individual contributor
 * Michael Ferguson, [HPE]
+* Rahul Ghangas, individual contributor
 * Ben Harshbarger, [HPE]
```

Comment:
@bradcray - merged a contribution from @SamuelHoward last week:


```suggestion
* Ben Harshbarger, [HPE]
* Samuel Howard,  individual contributor
```

Reply:
Got it, thanks!  (and thanks @SamuelHoward!)

Reply:
My pleasure!